src/pages/_components/GetStarted.astro

```diff
@@ -20,6 +20,8 @@ import houston from "../_assets/houston.png"
 			data-src={houston.src}
 			data-src-hover={houstonHappy.src}
 			src={houston.src}
+			width={houston.width}
```

Comment:
I've got a hunch webp would help with these images as well!  Same for lazy loading and async decoding, no reason to block on this image down the page

Reply:
Good hunch! These are 90% smaller with WebP 🎉